<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListControlWidget</class>
 <widget class="QWidget" name="ListControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="downButton">
     <property name="text">
      <string>Move Item Down</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="upButton">
     <property name="text">
      <string>Move Item Up</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>Delete Item</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset List</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>resetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
